### PR TITLE
feat: route53 dns service

### PR DESCRIPTION
**Pull request by websoftwares**
Hello everyone, 

I have added the route53 DNS service from AWS as I think it is widely used and could be handy for others.

Gr
Boris

### CODE REVIEW

File: src/dnslinkers/route53.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        ]
+      }
+    })
+
+    try {
+
+   await client.send(command)
+
+  } catch (err) {
+      console.log(err)
```
Comment (hacdias) [inline src/dnslinkers/route53.js:68]: We should just let it throw the error. It'll be handled by the caller.

**Review by hacdias — CHANGES_REQUESTED** [review]

**Comment by hacdias** [comment]
Very nice, could you just remove the try catch and making sure the code passes the linting?

File: src/dnslinkers/route53.js (hunk 1 above)
Comment (websoftwares) [reply src/dnslinkers/route53.js:68]: ah ok let me update that thanks!

**Comment by websoftwares** [comment]
@hacdias updated the PR with the changes requested.

**Review by hacdias — APPROVED** [review]

**Pull request merged**